runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -330,6 +323,19 @@
     @Getter
     private NodeRouterPool nodeRouterPool;
 
+    @AllArgsConstructor
+    public class TrimMarks {
+        public final long trimMark;
+        public final long trimTimestamp;
+    }
+
+    @Getter
+    private final LinkedList<TrimMarks> trimMarkList = new LinkedList<>();
+    public void addTrimMark(@NonNull long trimMark, @NonNull long timestamp) {
+        trimMarkList.addLast(new TrimMarks(trimMark, timestamp));
+    }
+    public volatile long matureTrimMark = Address.NON_ADDRESS;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make matureTrimMark a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)